browser-test/src/trusted_intermediary.test.ts

```diff
@@ -38,13 +38,38 @@ test.describe('Trusted intermediaries', () => {
       lastName: 'last',
       dobDate: '2021-12-12',
     }
-    await tiDashboard.expectSuccessAlert()
+    await tiDashboard.expectSuccessAlertOnUpdate()
     await validateScreenshot(page.locator('main'), 'edit-client-success-alert')
 
     await page.click('#ti-dashboard-link')
     await waitForPageJsLoad(page)
     await tiDashboard.expectDashboardContainClient(updatedClient)
   })
+  test('verify success toast screenshot on adding new client', async () => {
+    const {page, tiDashboard} = ctx
```

Comment:
nitpick: Gwen recently showed us how to include the `{page, tiDashboard}` in the test line:

`test('verify success toast screenshot on adding new client', async ({page, tiDashboard}) => {`

@gwendolyngoetz  do we want to start using that convention for all new tests?

Reply:
I committed support for using these custom fixtures earlier this week, but won't have time until tonight or this weekend to update the wiki and provide more detailed instructions on how to migrate.

It's fine to use them going forward, but it's best to convert an entire file all at once. Otherwise you run the risk of using the page from ctx (old way) with the page as a fixture (new way) which could result it weird test results.

Converting the entire file should be done as a separate PR on test files that are very large too.